<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_SortedListClass</class>
 <widget class="QWidget" name="W_SortedListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>W_SortedList</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="L_Name">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Stadium">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stadium</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Capacity">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Capacity</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Location">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Location</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_RoofType">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Roof Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Surface">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Surface</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Opened">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Opened</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Conference">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Conference</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_Division">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Division</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
